magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/LineLengthCheck.java

```diff
@@ -45,8 +54,8 @@ protected void walkPreMagik(final AstNode node) {
       for (int i = 0; i < line.length(); ++i) {
         final char chr = line.charAt(i);
         if (chr == '\t') {
-          final int mod = columnNo % TAB_WIDTH;
-          columnNo += TAB_WIDTH - mod;
+          final int mod = columnNo % tabWidth;
```

Comment:
Please use `this.tabWidth` for instance variables. This makes it easier to determine where the variable comes from.